src/BlazorWebView/src/Maui/IBlazorWebView.cs

```diff
@@ -15,6 +15,11 @@ public interface IBlazorWebView : IView
 		/// </summary>
 		string? HostPage { get; }
 
+		/// <summary>
+		/// Gets or sets the path for initial navigation within the Blazor navigation context when the Blazor component is finished loading.
+		/// </summary>
+		public string StartPath { get; set; }
```

Comment:
Is it possible to also set ` = "/"` on this? That would be even nicer because it would have the correct default value in other implementations not aware of this change. If the syntax doesn't allow it, it's fine.

Reply:
It is not possible, the syntax doesn't allow it.